app/controllers/catalog_controller.rb

```diff
@@ -103,7 +103,7 @@ def self.modified_field
       all_names = config.show_fields.values.map(&:field).join(" ")
       title_name = solr_name("title", :stored_searchable)
       field.solr_parameters = {
-        qf: "#{all_names} id file_format_tesim all_text_timv",
+        qf: "#{all_names} id file_format_tesim all_text_timv creator_tesim",
```

Comment:
Do we know what all_text_timv has in it?  Should we just be indexing alternate creator names into all_text_timv as well?